php72/Dockerfile

```diff
@@ -0,0 +1,149 @@
+    rpm -Uvh /usr/local/ius-release.rpm
+
+RUN yum -y install \
+    # Install PHP
+    php72 \
+    php72-php-devel \
+    php72-php-gd \
+    php72-php-xml \
+    php72-php-pdo \
+    php72-php-mysql \
+    php72-php-mysqlnd \
+    php72-php-mbstring \
+    php72-php-fpm \
+    php72-php-opcache \
+    php72-php-pecl-memcache \
+    php72-php-pecl-xdebug \
+    php72-php-posix \
+    php72-php-mcrypt \
+    php72-php-pecl-yaml \
+    php72-php-pecl-zip \
+    # Install Ruby
+    ruby193 \
+    ruby193-rubygems \
+    ruby193-ruby-devel \
+    # Install Miscellaneous Tools
+    bzip2 \
+    gcc-c++ \
+    git2u-all \
+    httpd-tools \
+    jq \
+    make \
+    mariadb \
+    nmap-ncat \
+    patch \
+    postgresql \
+    pv \
+    rsync \
+    sendmail \
+    unzip \
+    # Necessary for drush
+    which \
+    # Necessary library for phantomjs per https://github.com/ariya/phantomjs/issues/10904
+    fontconfig \
+    && yum clean all
+
+# Ensure ruby193 binaries are in path
+ENV PATH /root/.composer/vendor/bin:/opt/rh/ruby193/root/usr/bin:/usr/local/sbin:/usr/local/bin:/usr/sbin:/usr/bin:/sbin:/bin
+
+# Ensure PHP binaries are in path
+RUN ln -sfv /opt/remi/php72/root/usr/bin/* /usr/bin/ && \
+    ln -sfv /opt/remi/php72/root/usr/sbin/* /usr/sbin/
+
+# Install PHPRedis extension
+ENV PHPREDIS_VERSION 3.1.2
+RUN curl -L -o /tmp/phpredis.tar.gz "https://github.com/phpredis/phpredis/archive/$PHPREDIS_VERSION.tar.gz" && \
+    tar -xzf /tmp/phpredis.tar.gz -C /tmp && \
+    rm /tmp/phpredis.tar.gz && \
+    cd "/tmp/phpredis-$PHPREDIS_VERSION" && \
+    phpize && \
+    ./configure && \
+    make && \
+    make install
+
+# Enable other ruby193 SCL config
+ENV LD_LIBRARY_PATH /opt/rh/ruby193/root/usr/lib64
+ENV PKG_CONFIG_PATH /opt/rh/ruby193/root/usr/lib64/pkgconfig
+
+# Ensure $HOME is set
+ENV HOME /root
+
+# Configure Git
+# https://git-scm.com/docs/git-config#git-config-corepreloadIndex
+RUN git config --global core.preloadindex true
+
+# Install Composer
+RUN curl -sS https://getcomposer.org/installer | php && mv composer.phar /usr/bin/composer
+# https://getcomposer.org/doc/03-cli.md#composer-allow-superuser
+ENV COMPOSER_ALLOW_SUPERUSER 1
+
+# Add global composer dependencies.
+# Drush for drupal development.
+# prestissimo for parallelized composer download operations.
+RUN composer global require drush/drush:8.x hirak/prestissimo:^0.3 && \
+  composer clear-cache
+
+# Install nvm, supported node versions, and default cli modules.
+ENV NVM_DIR $HOME/.nvm
+ENV NODE_VERSION 4
+RUN (curl https://raw.githubusercontent.com/creationix/nvm/v0.33.2/install.sh | bash) && \
+  chmod +x $HOME/.nvm/nvm.sh
+
+# Node 4.x (LTS)
+RUN source $NVM_DIR/nvm.sh && \
+  nvm install 4 && \
+  npm install -g bower grunt-cli gulp-cli && \
+  npm cache clean
+# Node 6.x (stable)
+RUN source $NVM_DIR/nvm.sh && \
+  nvm install 6 && \
+  npm install -g bower grunt-cli gulp-cli && \
+  npm cache clean
+# Node 8.x (stable)
```

Comment:
Can we go as far as removing node 8?  I think if someone is using new/recent versions of PHP we should be willing to let go of old versions of node.

Reply:
Definitely not.  Node 8 is still the LTS supported version.  I think we should always support the LTS and Current here. But we have lots of projects on Node 8 who might still want/need php 7.2.